cypress/support/auth.ts

```diff
@@ -89,7 +89,7 @@ Cypress.Commands.add('edaLogin', () => {
         Cypress.env('EDA_PASSWORD') as string,
         'EDA server'
       );
-      cy.hasTitle(/^Welcome to EDA Server$/);
```

Comment:
curious as to why `EDA Server` has been removed here and `AWX` in the `awxLogin` command

Reply:
Because in the standalone version of EDA, it says "Welcome To Ansible Automation Platform", not "Welcome to EDA Server"